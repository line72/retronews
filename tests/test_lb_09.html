<blockquote>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
</blockquote>
<blockquote>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
<pre><code>def hello_world():
  print('hello')
  if True:
    print('world')
  print('!!!')

hello_world()
</code></pre>
</blockquote>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
<pre><code>def hello_world():
  print('hello')
  if True:
    print('world')
  print('!!!')

hello_world()
</code></pre>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
