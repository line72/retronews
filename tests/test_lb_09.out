> Lorem ipsum dolor sit amet, consectetur adipiscing elit

> Lorem ipsum dolor sit amet, consectetur adipiscing elit
> 
> | def hello_world():
> |   print('hello')
> |   if True:
> |     print('world')
> |   print('!!!')
> | 
> | hello_world()

Lorem ipsum dolor sit amet, consectetur adipiscing elit

| def hello_world():
|   print('hello')
|   if True:
|     print('world')
|   print('!!!')
| 
| hello_world()

Lorem ipsum dolor sit amet, consectetur adipiscing elit